packages/web3-providers/src/types.ts

```diff
@@ -113,6 +113,94 @@ export namespace RSS3BaseAPI {
         name: string
     }
 
+    export interface NFT_Contract {
+        address: string
+        name: string
+        symbol: string
+    }
+
+    export interface NFT_Traits {
+        trait_type: string
+        value: string
+    }
+    export interface NFT_Type {
+        asset_contract: NFT_Contract
+        chain: string
+        description: string
+        image_preview_url: string
+        image_preview_url_ct: string
+        image_thumbnail_url: string
+        image_thumbnail_url_ct: string
+        image_url: string
+        image_url_ct: string
+        name: string
+        received_at: string
+        token_id: string
+        traits: NFT_Traits[]
+    }
+
+    export interface NFT {
+        id: string
+        detail: NFT_Type
+    }
+
+    export interface DonationTx {
+        adminAddr: string
+        amount: string
+        approach: string
+        donor: string
+        formatedAmount: string
```

Comment:
`formated` is a typo, is it defined by RSS3? If so, then we can ignore it.

Reply:
yes, RSS3 api defined it